llmfoundry/utils/config_utils.py

```diff
@@ -161,6 +161,7 @@ class TrainConfig:
     load_strict_model_weights: bool = True
     load_ignore_keys: Optional[List[str]] = None
     save_ignore_keys: Optional[List[str]] = None
+    just_hf_checkpoint: bool = False
```

Comment:
```suggestion
    only_hf_checkpoint: bool = False
```
More consistent with naming elsewhere

Reply:
done